udata/core/dataset/apiv2.py

```diff
@@ -0,0 +1,151 @@
+    'deleted': fields.ISODateTime(description='The deletion date if deleted'),
+    'archived': fields.ISODateTime(description='The archival date if archived'),
+    'featured': fields.Boolean(description='Is the dataset featured'),
+    'private': fields.Boolean(
+        description='Is the dataset private to the owner or the organization'),
+    'tags': fields.List(fields.String),
+    'badges': fields.List(fields.Nested(badge_fields),
+                          description='The dataset badges',
+                          readonly=True),
+    'resources': fields.Raw(attribute=lambda o: {
+        'rel': 'subsection',
+        'href': url_for('apiv2.resources', dataset=o.id, page=1, page_size= 20, _external=True),
+        'type': 'GET',
+        'total': len(o.resources)
+        }, description='Link to the dataset resources'),
+    'community_resources': fields.Raw(attribute=lambda o: {
+        'rel': 'subsection',
+        'href': url_for('api.community_resources', dataset=o.id, page=1, page_size= 20, _external=True),
+        'type': 'GET',
+        'total': len(o.community_resources)
+        }, description='Link to the dataset community resources'),
+    'frequency': fields.String(
+        description='The update frequency', required=True,
+        enum=list(UPDATE_FREQUENCIES), default=DEFAULT_FREQUENCY),
+    'frequency_date': fields.ISODateTime(
+        description=('Next expected update date, you will be notified '
+                     'once that date is reached.')),
+    'extras': fields.Raw(description='Extras attributes as key-value pairs'),
+    'metrics': fields.Raw(attribute=lambda o: o.get_metrics(), description='The dataset metrics'),
+    'organization': fields.Nested(
+        org_ref_fields, allow_null=True,
+        description='The producer organization'),
+    'owner': fields.Nested(
+        user_ref_fields, allow_null=True, description='The user information'),
+    'temporal_coverage': fields.Nested(
+        temporal_coverage_fields, allow_null=True,
+        description='The temporal coverage'),
+    'spatial': fields.Nested(
+        spatial_coverage_fields, allow_null=True,
+        description='The spatial coverage'),
+    'license': fields.String(attribute='license.id',
+                             default=DEFAULT_LICENSE['id'],
+                             description='The dataset license'),
+    'uri': fields.UrlFor(
+        'api.dataset', lambda o: {'dataset': o},
+        description='The dataset API URI', required=True),
+    'page': fields.UrlFor(
+        'datasets.show', lambda o: {'dataset': o},
+        description='The dataset page URL', required=True, fallback_endpoint='api.dataset'),
+    'quality': fields.Raw(description='The dataset quality', readonly=True),
+    'last_update': fields.ISODateTime(
+        description='The resources last modification date', required=True),
+})
+
+
+resource_page_fields = apiv2.model('ResourcePage', {
```

Comment:
This might be useless as we return an explicit json from the endpoint. Using the `fields.pager` like done with the community resources does not work as we do not return a mongoengine cursor.